grafana/resource_cloud_plugin_test.go

```diff
@@ -0,0 +1,82 @@
+package grafana
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceCloudPluginInstallation(t *testing.T) {
+	CheckCloudAPITestsEnabled(t)
+
+	slug := "terraformprovidergrafana"
+	pluginSlug := "aws-datasource-provisioner-app"
+	pluginVersion := "1.7.0"
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGrafanaCloudPluginInstallation(slug, pluginSlug, pluginVersion),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCloudPluginInstallationCheckExists("grafana_cloud_plugin_installation.test-installation", slug, pluginSlug),
+					resource.TestCheckResourceAttrSet("grafana_cloud_plugin_installation.test-installation", "id"),
+					resource.TestCheckResourceAttr("grafana_cloud_plugin_installation.test-installation", "stack_slug", slug),
+					resource.TestCheckResourceAttr("grafana_cloud_plugin_installation.test-installation", "slug", "aws-datasource-provisioner-app"),
+					resource.TestCheckResourceAttr("grafana_cloud_plugin_installation.test-installation", "version", "1.7.0")),
+			},
```

Comment:
You have an importer so an import test would be nice. Example: https://github.com/grafana/terraform-provider-grafana/blob/master/grafana/resource_folder_test.go#L41-L45